File: LUA/Cap05/c05ex07.lua
-- c05ex07.lua

i = 1
repeat
  io.write("Entre um valor numerico: ")
  n = tonumber(io.read())

  r = n * 3

  print("Resultado = " .. r)
  print()

  i = i + 1
until not (i <= 5)

print()
io.write("Tecle <Enter> para encerrar... ")
io.read()

File: VB_NET/Cap05/c05ex07.vb
' c05ex07.vb

Imports System
Imports System.Console

Module Program
    Sub Main(args As String())
        Dim I As Integer
        Dim N As Integer
        Dim R As Integer

        I = 1
        Do 
            Write("Entre um valor numerico: ")
            N = Integer.Parse(ReadLine())

            R = N * 3

            WriteLine("Resultado = " & R & vbCrLf)

            I += 1
        Loop While (I <= 5)

        Write(vbCrLf & "Tecle <Enter> para encerrar... ")
        ReadLine()
    End Sub
End Module
